ql/src/java/org/apache/hadoop/hive/ql/optimizer/calcite/rules/HiveRewriteToDataSketchesRules.java

```diff
@@ -68,25 +82,32 @@
  *       ⇒ SELECT ds_kll_quantile(ds_kll_sketch(CAST(id AS FLOAT)), 0.2) FROM sketch_input;
  *    </pre>
  *  </li>
+ *  <li>{@code cume_dist() over (order by id)}
```

Comment:
I think these apidoc could be moved to the rewrite-rules - but they also have there meaning here as well...maybe move them and add a more brief description here?